code/game/sound.dm

```diff
@@ -111,7 +111,7 @@
 /client/proc/playcreditsmusic(vol = 85) // FULPSTATION
 	set waitfor = FALSE
 	if(prefs && (prefs.toggles & SOUND_LOBBY))
-		SEND_SOUND(src, sound('sound/Fulpsounds/Fulp_Piano.ogg', repeat = 0, wait = 0, volume = vol, channel = CHANNEL_LOBBYMUSIC)) // FULPSTATION: Play end music! SO SAD!
+		SEND_SOUND(src, pick(sound('sound/Fulpsounds/Fulp_Piano.ogg', repeat = 0, wait = 0, volume = vol, channel = CHANNEL_LOBBYMUSIC),sound('sound/Fulpsounds/Fulp_Uhoh_Stinky.ogg', repeat = 0, wait = 0, volume = vol, channel = CHANNEL_LOBBYMUSIC))) // FULPSTATION: Play end music! SO SAD!
```

Comment:
we may at some point want to refactor this into a config item so we can add files and throw em in the round end shuffle